components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/basicauth/BasicAuthAuthenticator.java

```diff
@@ -139,6 +154,8 @@ public AuthenticationResponse authenticate(MessageContext synCtx) {
                     verbInfoDTO.setAuthType(APIConstants.AUTH_NO_AUTHENTICATION);
                 }
                 verbInfoDTO.setThrottling(operationThrottlingMappingList.get(operation));
+                verbInfoDTO.setRequestKey(apiContext + "/" + apiVersion + operation + ":" + httpMethod);
```

Comment:
use constants for hard coded strings

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9250